
  File "script.py", line 25
SyntaxError: Non-ASCII character '\xe2' in file script.py on line 25, but no encoding declared; see http://python.org/dev/peps/pep-0263/ for details
